src/main/java/org/jruby/ext/openssl/OCSPBasicResponse.java

```diff
@@ -0,0 +1,147 @@
+/*
+* The contents of this file are subject to the Common Public License Version 1.0
+* (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.eclipse.org/legal/cpl-v10.html
+* 
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+* FOR APARTICULAR PURPOSE AND NONINFRINGEMENT.
+* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+* DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+* ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+* DEALINGS IN THE SOFTWARE.
+*
+*  Copyright (C) 2007-2009 Ola Bini <ola.bini@gmail.com>
```

Comment:
would prefer to have your name here instead of Ola's or just the `(C) 2017 The JRuby Team` if your shy :)